view_w.php

```diff
@@ -39,7 +39,7 @@
 $nextStr = translate ( 'Next' );
 $prevStr = translate ( 'Previous' );
 
-$can_add = ( empty ( $ADD_LINK_IN_VIEWS ) || $ADD_LINK_IN_VIEWS != 'N' );
+$can_add = ( $ADD_LINK_IN_VIEWS === 'Y' );
```

Comment:
Instead of $Id$ we usually use the commit id.  If you do a git log on a particular file, you will see the history and commit ids.  So, I don't see much point to retaining the old CVS $Id$ stuff.  To remove you could just revert the change and commit it as it was before.  I believe that changes is related to the one that switches to using '==='.